HS9_py2/heatsource9/Dieties/IniParamsDiety.py

```diff
@@ -5,6 +5,11 @@
 """
 from collections import defaultdict
 
+try:
+  basestring
+except NameError:
+  basestring = str
+  

```

Comment:
Begin cleaning up for Python3 migration. This was necessary to implement initial tests.